roles/adminconsole/ac_configAc/tasks/main.yaml

```diff
@@ -89,11 +94,11 @@
     - { regexp: "{{ placeHolder_clientId }}", line: "{{ ac_client_id }}" }
     - { regexp: "{{ placeHolder_logLevel }}", line: "{{ ac_log_level }}" }
     - { regexp: "{{ placeHolder_tokenSecret }}", line: "{{ ac_token_secret }}" }
-    - { regexp: "{{ placeHolder_sslVerifyCerts }}", line: "{{ ac_ssl_verify_certs }}" }
-    - { regexp: "{{ placeHolder_sslHostnameValidation }}", line: "{{ ac_ssl_hostname_validation }}" }
-    - { regexp: "{{ placeHolder_sslCaPath }}", line: "{{ ac_ssl_ca_path }}" }
-    - { regexp: "{{ placeHolder_sslCertPath }}", line: "{{ ac_ssl_cert_path }}" }
-    - { regexp: "{{ placeHolder_sslKeyPath }}", line: "{{ ac_ssl_key_path }}" }
+    - { regexp: "{{ placeHolder_sslVerifyCerts }}", line: "{% if enable_brkr_tls is defined and enable_brkr_tls|bool %}false{% else %}true{% endif %}" }
+    - { regexp: "{{ placeHolder_sslHostnameValidation }}", line: "{% if enable_brkr_tls is defined and enable_brkr_tls|bool %}false{% else %}true{% endif %}" }
+    - { regexp: "{{ placeHolder_sslCaPath }}", line: "{% if enable_brkr_tls is defined and enable_brkr_tls|bool %}{{ tgt_adminConsole_sec_dir }}/tls/rootca/{{ public_cacert_name }}{% else %}{% endif %}" }
```

Comment:
useless {% else %} block

Reply:
Regarding "useless {% else %} block", first, the method is copied from original placeholder  "clientToken", and "caCertificate".  Second, if the double quote is in place of the filed in the local.json, then it will be "" and it will clear the value in the field in default.json.  if the no double quote is in place of the filed in the local.json, then it will be  nothing and it will not clear the value in the field in default.json.

Reply:
ok